tests/bluetooth/tester/src/btp/btp_bap.h

```diff
@@ -6,22 +6,20 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-
-#define PACS_CHARACTERISTIC_SINK_PAC			0x01
-#define PACS_CHARACTERISTIC_SOURCE_PAC			0x02
-#define PACS_CHARACTERISTIC_SINK_AUDIO_LOCATIONS	0x03
-#define PACS_CHARACTERISTIC_SOURCE_AUDIO_LOCATIONS	0x04
-#define PACS_CHARACTERISTIC_AVAILABLE_AUDIO_CONTEXTS	0x05
-#define PACS_CHARACTERISTIC_SUPPORTED_AUDIO_CONTEXTS	0x06
-
-
 /* PACS commands */
-#define PACS_READ_SUPPORTED_COMMANDS	0x01
-struct pacs_read_supported_commands_rp {
+#define BTP_PACS_READ_SUPPORTED_COMMANDS			0x01
+struct btp_pacs_read_supported_commands_rp {
 	uint8_t data[0];
 } __packed;
 
-#define PACS_UPDATE_CHARACTERISTIC	0x02
-struct pacs_update_characteristic_cmd {
+#define BTP_PACS_CHARACTERISTIC_SINK_PAC			0x01
+#define BTP_PACS_CHARACTERISTIC_SOURCE_PAC			0x02
+#define BTP_PACS_CHARACTERISTIC_SINK_AUDIO_LOCATIONS		0x03
+#define BTP_PACS_CHARACTERISTIC_SOURCE_AUDIO_LOCATIONS		0x04
+#define BTP_PACS_CHARACTERISTIC_AVAILABLE_AUDIO_CONTEXTS	0x05
+#define BTP_PACS_CHARACTERISTIC_SUPPORTED_AUDIO_CONTEXTS	0x06
```

Comment:
I appreciate that this is how it was done before, and how `hci.h` also does it, but we could also consider putting the commands into an `enum` as recommended by https://www.kernel.org/doc/html/v4.10/process/coding-style.html#macros-enums-and-rtl

Reply:
> as comparing value outside of enum specified range is not defined

I'm not sure I follow that; can you elaborate?

Reply:
Those structs are used to parse incoming binary data from BTP, so lets say we have this:

enum BTP_PACS_CHR __packed {
   BTP_PACS_CHARACTERISTIC_SINK_PAC = 0x01,
   BTP_PACS_CHARACTERISTIC_SOURCE_PAC = 0x02,
}

struct btp_pacs_cmd {
  enum BTP_PACS_CHR chr;
}

and then we use it in handler cb to parse 1 byte buffer, where cmd[0] = 0x42
static uint8_t pacs_cmd(const void *cmd, uint16_t cmd_len,  void *rsp, uint16_t *rsp_len)
{
   const struct btp_pacs_cmd *cp = cmd;

   so now cp->chr has bianry value of 0x42 which is outside of defined enum range
}

Reply:
but this is the case exactly I'm not liking about using enums for this,    doing switch on enum while value is outside of valid range